src/views/ProblemDefinition.tsx

```diff
@@ -67,83 +65,109 @@ function ProblemDefinition({
     objective_names: [],
     variables: [[]],
     variable_names: [],
-    minimize: []
+    minimize: [],
   });
-  const [discreteProblemLoaded, SetDiscretePoblemLoaded] = useState<boolean>(false);
+  const [discreteProblemLoaded, SetDiscretePoblemLoaded] =
+    useState<boolean>(false);
 
-  const onDrop = useCallback((acceptedFiles: File[]) => {
-    const file = acceptedFiles[0]; // only single files are handled
-    const name = file.name;
-    const reader = new FileReader()
+  const onDrop = useCallback(
+    (acceptedFiles: File[]) => {
+      const file = acceptedFiles[0]; // only single files are handled
+      const name = file.name;
+      const reader = new FileReader();
 
-    reader.onabort = () => console.log('file reading was aborted')
-    reader.onerror = () => console.log('file reading has failed')
-    reader.onload = () => {
-      const textStr = reader.result as string
-      parse(textStr, { comment: '#', trim: true, skipEmptyLines: true }, (err, output) => {
-        if (err != undefined) {
-          SetMessage(`Error parsing csv file: ${err.message}`);
-          return;
-        } else {
-          // Output should contain at least 3 rows
-          if (output.length < 3) {
-            SetMessage("The csv data should contain at least three rows. Less than three rows provided.");
```

Comment:
most of the changes highlighted by diff are just because of code formatting. I don't think I have made any changes to this file except adding support for a special Problem class for IOPIS